website/guides/advanced/protobuf.md.erb

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Given that we noted some issues with Vector splitting proto events, I want to move this to an separate PR.